Invoke-Locksmith.ps1

```diff
@@ -106,7 +106,7 @@ foreach($certpublishersSID in $AllDomainsCertPublishersSIDs) {
         $SafeUsers += '|' + $_.SID
     }
 }
-foreach($domainadminsSID in $AllDomainsDomainAdminsSIDs) {
+foreach($domainadminsSID in $AllDomainsDomainAdminSIDs) {
```

Comment:
This is actually the one line where I spelled it right. If you modify lines 68 and 96 to match this line, I'll accept the PR. Thanks!

Reply:
Done ✔️